packages/backend/src/index.ts

```diff
@@ -49,9 +49,9 @@ function makeCreateEnv(loadedConfigs: AppConfig[]) {
     const logger = getRootLogger().child({ type: 'plugin', plugin });
     // Supported DBs are sqlite and postgres
     const isPg = [
-      'POSTGRES_USER',
-      'POSTGRES_HOST',
-      'POSTGRES_PASSWORD',
+      'POSTGRES.USER',
+      'POSTGRES.HOST',
+      'POSTGRES.PASSWORD',
```

Comment:
Let's make these lowercase while we're at it; looks nicer.

And now you just need to check for the presence of the parent `postgres` key, not all of the children.

Reply:
Another approach would be to do something like https://github.com/spotify/backstage/issues/1882#issuecomment-671289593, which would have a specific `client` key to indicate which database to use.